lib/pbench/test/functional/server/test_datasets.py

```diff
@@ -361,17 +388,21 @@ def test_details(self, server_client: PbenchServerClient, login_user):
         )
         for d in datasets:
             print(f"\t... checking run index for {d.name}")
+            indexed = not d.metadata["server.archiveonly"]
             response = server_client.get(
-                API.DATASETS_DETAIL, {"dataset": d.resource_id}
+                API.DATASETS_DETAIL, {"dataset": d.resource_id}, raise_error=False
             )
             detail = response.json()
-            indexed = not d.metadata["server.archiveonly"]
             if indexed:
+                assert (
+                    response.ok
+                ), f"DETAILS for {d.name} failed with {detail['message']}"
```

Comment:
Should there be a colon here?

Reply:
I suppose I could have used a ":" instead of "with", and we tend to do that in our production code messages so I suppose there's a "consistent" argument here. On the other hand, this is just a debugging message when a functional test case fails so it doesn't seem worth changing.